crates/snapbox/src/data.rs

```diff
@@ -12,6 +12,12 @@ enum DataInner {
     Text(String),
 }
 
+#[derive(Clone, Debug, PartialEq, Eq)]
```

Comment:
What about `Copy` and `Hash`?

Reply:
I was copying the other derives in the file. I will add `Copy` and `Hash`